Framework/Beamline/inc/MantidBeamline/ComponentInfo.h

```diff
@@ -105,8 +109,9 @@ class MANTID_BEAMLINE_DLL ComponentInfo {
   ComponentInfo &operator=(const ComponentInfo &other) = delete;
   /// Clone method
   std::unique_ptr<ComponentInfo> cloneWithoutDetectorInfo() const;
-  std::vector<size_t> detectorsInSubtree(const size_t componentIndex) const;
+  std::vector<size_t> detectorsInFullSubtree(const size_t componentIndex) const;
```

Comment:
Why is this renamed? What is a full subtree in comparison to a subtree?

Reply:
This is to reflect the fact that all detectors below this are returned.  For example, doing this on the root will return all detectors in the instrument.

Reply:
That is implied by the word `tree`, isn't it?